tests/config/data/make-test-rocrates.py

```diff
@@ -134,10 +139,26 @@ def patch_metadata_graph_node(metadata_file, node, properties):
                               'name': 'sort-and-change-case-invalid-service-type'
                           })
 
+# GitHub Actions crate
+patch_metadata_graph_node('crates/ro-crate-galaxy-sortchangecase-github-actions/ro-crate-metadata.json',
+                          node=("@type", "TestInstance"),
+                          properties={
+                              'url': 'https://api.github.com',
+                              'resource': '/repos/crs4/life_monitor/actions/workflows/4094661',
```

Comment:
`resource` is a relative path in Jenkins and Travis instances. We need to stick to the same convention here.

Reply:
Done in ac54584